db/migrate/20240409222653_backfill_projects_latest_version_id.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class BackfillProjectsLatestVersionId < ActiveRecord::Migration[7.0]
+  def up
+    # This migration has already been run manually in production.
+    return if Rails.env.production?
+
+    Project
+      .in_batches(of: 10_000) do |projects|
+        sql = <<~SQL
```

Comment:
🎱 I wonder if will need to get/set this for multiple projects at a time in biz logic, in which case it might be useful to have this in a class method or interaction instead of just in a backfill. Then it can also be tested via the usual routes.